rosbag2_tests/test/rosbag2_tests/test_rosbag2_record_end_to_end.cpp

```diff
@@ -803,3 +664,8 @@ TEST_F(RecordFixture, rosbag2_record_and_play_multiple_topics_with_filter) {
   // stops thread
   sub->add_subscription<test_msgs::msg::Strings>(first_topic_name, 0);
 }
+
+INSTANTIATE_TEST_SUITE_P(
+  TestRecordEndToEnd,
+  RecordFixture,
+  ::testing::Values(std::string("sqlite3")));
```

Comment:
@emersonknapp Curious about if can already add "mcap" as parameter option?
i.e. `::testing::Values(std::string("sqlite3"), std::string("mcap")));`

Reply:
Almost, but not quite yet. There is still a circular dependency in tests being fixed in `rosbag2_storage_mcap` before we can activate parameters for all tests.

I'd like to move forward with this PR, and we can start parameterizing all `rosbag2_tests` and `rosbag2_cpp` tests for both mcap and sqlite in a followup.